_pkgdown.yml

```diff
@@ -32,3 +32,4 @@ reference:
       - modify_zero_recode
       - add_blank_row
       - roche_pvalue
+      - roche_percent
```

Comment:
lets document the function with `roche_pvalue` in a file named `label_roche.R`

Reply:
I've updated the rdname to label_roche